common/source/docs/common-esc-telem-based-notch.rst

```diff
@@ -42,9 +42,10 @@ Once the notch filter(s) are setup, the effectiveness of them can be checked by
 
 While the log analysis of noise frequencies is not absolutely required prior to notch setup for RPM based mode, the logging and analysis done for the Confirmation flight using that method can be done in order to confirm the noise elimination, if desired.
 
-Dynamic Harmonics
-=================
+Multi-Source
+============
 
-By default the ESC based harmonic notch will use an average of the individual motor frequencies in order to drive the center frequency of the notch, this average is then used as the first harmonic and other harmonic notches are added at higher frequency multiples of the first harmonic. It is also possible to configure the harmonics to instead be first harmonics per motor. This gives four notches - one for each motor - that exactly tracks the motor speed. In dynamic flight this can give much better noise attenuation.
+By default the ESC based harmonic notch will use an average of the individual motor frequencies in order to drive the center frequency of the notch, this average is then used as the first harmonic and other harmonic notches are added at higher frequency multiples of the first harmonic. It is possible to configure the harmonics to instead be first harmonics per motor. This gives four notches - one for each motor - that exactly tracks the motor speed. In dynamic flight this can give much better noise attenuation.
+
+To configure this option set :ref:`INS_HNTCH_OPTS <INS_HNTCH_OPTS>` and/or :ref:`INS_HNTC2_OPTS <INS_HNTC2_OPTS>` to "2". You also need to reduce the bandwidth of the notch filter to around 1/4 of :ref:`INS_HNTCH_FREQ<INS_HNTCH_FREQ>`, otherwise you will likely have too high phase lag leading to worse PID performance. See the note above for more details.
```

Comment:
what if this is a hex???this assumes a quad...maybe should generalize based on motor count?

Reply:
Updated based on discussion on the Harmonic notch changes thread in Discord.